platform-api/che-core-api-infrastructure-local/src/main/java/org/eclipse/che/api/local/storage/LocalStorage.java

```diff
@@ -45,10 +47,13 @@
     private Gson gson;
 
     public LocalStorage(String rootDirPath, String fileName) throws IOException {
-        this(rootDirPath, fileName, Collections.emptyMap());
+        this(rootDirPath, fileName, emptyMap(), emptyMap());
     }
 
-    public LocalStorage(String rootDirPath, String fileName, Map<Class<?>, Object> typeAdapters) throws IOException {
+    public LocalStorage(String rootDirPath,
```

Comment:
Don't change this contrcutor, add a new one, it will help you to avoid all these unnecessary changes